app/models/player.rb

```diff
@@ -2,9 +2,6 @@ class Player < ApplicationRecord
   has_many :team_players
   has_many :teams, through: :team_players
 
-  # this method is really big, but I think I need all of it so it
```

Comment:
I looks like you might be able to use a case/when statement here since there are a lot of conditions, but not much logic for each of those conditions